src/api.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
```

Comment:
Really nice job here. I would also look in the notes at creating a custom Axios instance to save you having to repeat the base URL of your BE every time

Reply:
I would second this! ^^ look at axios.create